ccd-definition/CaseEventToComplexTypes/placement/PlacementConfidentialDocument-prepare-for-hearing.json

```diff
@@ -1,32 +0,0 @@
-[
-  {
-    "LiveFrom": "01/01/2017",
-    "ID": "PlacementConfidentialDocument",
-    "CaseEventID": "placement-PREPARE_FOR_HEARING",
```

Comment:
there is still  
{
...
    "CaseEventID": "placement-PREPARE_FOR_HEARING",
    "UserRole": "[LASOLICITOR]",
    "CRUD": "CRU"
  },